apps/docs/spec/supabase_dart_v2.yml

```
@@ -4683,6 +4683,56 @@ functions:
             }
           ]
           ```
+      - id: order-parent-table-by-a-referenced-table
+        name: Order parent table by a referenced table
+        code: |
+          ```dart
+          final data = await supabase
+              .from('cities')
+              .select('''
+                name,
+                country:countries (
+                  name
+                )
+              ''')
+              .order('country(name)', ascending: true)
```

Dart requires the comma at the end.
```suggestion
              .order('country(name)', ascending: true);
```

---

🙏 thanks!